src/readonce.py

```diff
@@ -60,6 +60,10 @@ def __reset_is_consumed(cls):
     def __update_is_consumed(cls):
         cls.__is_consumed = True
 
+    @classmethod
+    def __is_reset(cls):
+        return len(cls.__secrets) == 0 and cls.__is_consumed is False
```

Comment:
The idea of simple function is great and neat thanks for your time.
But, I would like to leave as `not cls.secrets and not cls.__is_consumed`
Could you please kindly change this one?
Thanks.

Reply:
Changed the conditions. See in the next commit.